src/frontend/src/components/FilePicker/index.tsx

```diff
@@ -61,9 +61,9 @@ export default function FilePicker({
         sdsType="primary"
         sdsStyle="square"
         onClick={shouldConfirm ? openDialog : handleClick}
-        disabled={isDisabled}
+        disabled={isLoading}
```

Comment:
Currently the loading state is the only time the button is disabled.  If we want to separate this functionality in the future, we can add a separate "isDisabled" prop.